packages/wallet/frontend/src/pages/index.tsx

```diff
@@ -124,28 +131,47 @@ const HomePage: NextPageWithLayout<HomeProps> = ({ accounts, user }) => {
 
 export const getServerSideProps: GetServerSideProps<{
   accounts: Account[]
+  incomingOutgoingBalances: Record<string, number>
   user: {
     firstName: string
     lastName: string
     email: string
   }
 }> = async (ctx) => {
-  const response = await accountService.list(ctx.req.headers.cookie)
-  const user = await userService.me(ctx.req.headers.cookie)
+  const accountsResponse = await accountService.list(ctx.req.headers.cookie)
+  const userResponse = await userService.me(ctx.req.headers.cookie)
 
-  if (!response.success || !user.success) {
+  if (
+    !accountsResponse.success ||
+    !userResponse.success ||
+    !accountsResponse.result
```

Comment:
At a quick glance, `accountResponse.result` might end up being an empty list:
<img width="107" alt="image" src="https://github.com/user-attachments/assets/191d5461-1cd7-408c-ba12-e0a425bb08e5">

This might not be the desired outcome of the condition^^
Double check if this can be the case, and if such, check for `accountsResponse.result.length` to reach desired condition